contracts/L2ForkArbitrator.sol

```diff
@@ -0,0 +1,189 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.20;
+
+// Allow mixed-case variables for compatibility with reality.eth, eg it uses question_id not questionId
```

Comment:
No strong opinion,  but I would rather keep consistency within this repo, then the consistency with other repo. 
I think techincally, its not need write question_id, right?

Reply:
Technically it'll work, it just means that when you compare this contract with previous audited code that does the same thing in a different context, you won't be able to use normal tools like `diff` because we made everything spuriously different (for consistency!)

Reply:
Okay, I don't mind too much